packages/@react-aria/dnd/package.json

```diff
@@ -26,7 +26,6 @@
     "@react-aria/utils": "^3.13.3",
     "@react-aria/visually-hidden": "^3.4.1",
     "@react-stately/dnd": "3.0.0-alpha.10",
```

Comment:
Was doing some auditing and have a clarifying question: should things like `@react-stately/dnd`/`@react-types/*` be in devDeps instead if they are only used to import types/interfaces? I was under the impression that we could do a `import type` for things like that and thus avoid needing to include them within the dependencies because they only matter during dev (aka typechecking)